_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -9,84 +9,297 @@ nav_order: 50
+
+#### Query examples
+
+The following example queries show how you can use the alerting query structure to monitor various metrics and conditions in your OpenSearch data. By adjusting the trigger conditions, input queries, and actions, you can create tailored alerting rules to suit your specific monitoring needs.
+
+**High CPU usage alert**
+```json
+{
+  "name": "High CPU Usage Alert",
+  "description": "Alerts when the average CPU usage exceeds 80% in the last hour",
+  "schedule": {
+    "interval": {
+      "period": 1,
+      "unit": "HOURS"
+    }
+  },
+  "trigger": {
+    "condition": {
+      "script": {
+        "source": "
+          double avgCpuUsage = ctx.payload.aggregations.avg_cpu_usage.value;
+          return avgCpuUsage > 80;
+        "
+      }
+    }
+  },
+  "input": {
+    "search": {
+      "request": {
+        "indices": ["system-metrics"],
+        "body": {
+          "size": 0,
+          "aggs": {
+            "avg_cpu_usage": {
+              "avg": {
+                "field": "cpu_usage"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "actions": [
+    {
+      "name": "Notify on Slack",
+      "type": "slack",
+      "slack": {
+        "message": "High CPU usage detected: {{ctx.payload.aggregations.avg_cpu_usage.value}}%"
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+**Low disk space alert**
+```json
+{
+  "name": "Low Disk Space Alert",
+  "description": "Alerts when the available disk space falls below 20%",
+  "schedule": {
+    "interval": {
+      "period": 30,
+      "unit": "MINUTES"
+    }
+  },
+  "trigger": {
+    "condition": {
+      "script": {
+        "source": "
+          double diskUsagePercentage = ctx.payload.aggregations.disk_usage_percentage.value;
+          return diskUsagePercentage > 80;
+        "
+      }
+    }
+  },
+  "input": {
+    "search": {
+      "request": {
+        "indices": ["system-metrics"],
+        "body": {
+          "size": 0,
+          "aggs": {
+            "disk_usage_percentage": {
+              "script": {
+                "source": "
+                  double totalDiskSpace = doc['disk_total'].value;
+                  double usedDiskSpace = doc['disk_used'].value;
+                  return (usedDiskSpace / totalDiskSpace) * 100;
+                "
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "actions": [
+    {
+      "name": "Notify on Email",
+      "type": "email",
+      "email": {
+        "to": ["ops@example.com"],
+        "subject": "Low Disk Space Alert",
+        "message": "Available disk space is below 20%: {{ctx.payload.aggregations.disk_usage_percentage.value}}%"
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+## Viewing alerting events
+
+To view alerting events, follow these steps:
+
+1. Open your local instance of OpenSearch Dashboards. This is typically `http://localhost:5601/app/home#/`.
```

Comment:
The URL listed here isn't necessarily correct. That URL would indicate that the frontend is running on the local machine, which probably isn't going to be the case. Is the URL example needed, or would it make more sense to say something like this?

> 1. Visit the OpenSearch Dashboards page for your cluster. To learn more about installing OpenSearch Dashboards, see [this page](https://opensearch.org/docs/latest/install-and-configure/install-dashboards/index/).

Reply:
Removed the URL